docs/sphinx/examples/python/bernstein_vazirani.py

```diff
@@ -94,6 +99,7 @@ def bernstein_vazirani(qubit_count: int):
     # a long time for the CPU-only backend to simulate 28+ qubits!
 
     qubit_count = args.size
+    random.seed(args.seed)
```

Comment:
I think it is nice if the simulation is random unless a random seed is explicitly provided.
```suggestion
    if args.seed != 0:
        random.seed(args.seed)
```